src/lib/utils/thorchain/memo/assertAndProcessMemo.test.ts

```diff
@@ -585,11 +585,11 @@ describe('assertAndProcessMemo', () => {
     it('should throw on invalid min out', () => {
       let memo =
         '$+:ETH.DAI-0X6B175474E89094C44DA98B954EEDEAC495271D0F:0x782C14C79945caD46Fbea57bb73d796366e76147::ss:50'
-      expect(() => assertAndProcessMemo(memo)).not.toThrow()
+      expect(() => assertAndProcessMemo(memo)).toThrow()
 
       memo =
         '$+:ETH.DAI-0X6B175474E89094C44DA98B954EEDEAC495271D0F:0x782C14C79945caD46Fbea57bb73d796366e76147:0:ss:50'
-      expect(() => assertAndProcessMemo(memo)).not.toThrow()
+      expect(() => assertAndProcessMemo(memo)).toThrow()
```

Comment:
yikes

Reply:
That wasn't a mistake, we previously inverted this on purpose! https://github.com/shapeshift/web/pull/6805